sandbox/grist/functions/lookup.py

```diff
@@ -153,7 +153,7 @@ def VLOOKUP(table, **field_value_pairs):
 
 class _Contains(namedtuple("_Contains", "value")):
   """
-  Use this marker with `Table.lookupRecords` to find records
+  Use this marker with [UserTable.lookupRecords](#lookuprecords) to find records
   where a column contains the given value, e.g:
```

Comment:
Here, it would be good to clarify "column" too. How about "records where a field of a list type (such as `Choice List` or `Reference List`) contains the given value"

Reply:
updated in second commit